backend/src/main/services/TaskService.ts

```diff
@@ -17,7 +17,7 @@ export class TaskService {
     if (TaskService.isThereNullValueTaskCreationDTO(taskDTO)) {
       throw new HttpException(StatusCodes.BAD_REQUEST, 'Request data is missing some values');
     }
-    return await this.taskRepository.create(taskDTO);
+    return this.taskRepository.create(taskDTO);
```

Comment:
Did you try running it? Does it break the code when you remove the await?

Reply:
This shouldn't break it. Since it return a promise. And for other services, we return it this way.